test/integration/param_parent_element.cc

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <string>
+#include <gtest/gtest.h>
+#include <ignition/math/Pose3.hh>
+
+#include "sdf/Element.hh"
+#include "sdf/Param.hh"
+#include "test_config.h"
+
+//////////////////////////////////////////////////
+TEST(ParamParentElement, SettingParentElement)
```

Comment:
Would these tests fit better in `Param_TEST.cc`?

Reply:
That makes sense, thanks! https://github.com/ignitionrobotics/sdformat/pull/589/commits/aa2a4d9d6e041b2485acade1422eb6680a90ee7b